### ISSUE TITLE
[New/Revise] simpify title of Scenario.trend() figure

### ISSUE THREAD

**Issue by lisphilar** [lisphilar/covid19-sir#687]
## Summary of this new feature
Revise the title of `Scenario.trend()` figure to simplify it as follows.
Before: "Japan: S-R trend changed on 19Feb2020,......18Mar2021"
After: "Japan: phases detected with S-R trend analysis"

Japan data with 37 phases at the current version 2.17.0-mu:
![trend_12_Japan](https://user-images.githubusercontent.com/7270139/112714037-b34d2b00-8f1b-11eb-8a53-dcc0707e3716.png)

The locations of labels will be also updated.

Codes:

```Python
import covsirphy as cs
data_loader = cs.DataLoader()
jhu_data = data_loader.jhu()
population_data = data_loader.population()
oxcgrt_data = data_loader.oxcgrt()
country="Japan"
snl = cs.Scenario(country=country, province=None)
snl.register(jhu_data, population_data, extras=[oxcgrt_data])
snl.trend()
```

**Comment by lisphilar** [comment]
This is relatec to #670.

**Comment by lisphilar** [comment]
Change points are listed in "Start" column of `Scenario.summary()` dataframe.
